source/_docs/automation/troubleshooting.markdown

```diff
@@ -3,41 +3,70 @@ title: "Troubleshooting automations"
 description: "Tips on how to troubleshoot your automations."
 ---
 
+Automations and {% term scripts %} can be debugged in a few different ways. You can [test run](#testing-your-automation) the full sequence of actions, or test each condition and action separately. [Traces](#traces) let you see details of every step after an automation is run. For complicated automations with {% term templates %}, see the section [testing templates](#testing-templates).
+
+## Testing your automation
+
+Many automations can be tested directly in the automation editor UI.
+
+### Running the entire automation
+
+In the 3-dots menu in the automation list or automation editor UI, click the **Run** button. This will execute all of the {% term actions %}, skipping all {% term triggers %} and {% term conditions %}. This lets you test the full sequence of actions, as if the automation was triggered and all conditions were true. Note that any [trigger ID](/automation/trigger/#trigger-id) used in your triggers will not be active when you test this way. The Trigger ID or any data passed by in the `trigger` data in conditions or actions can't be tested directly this way.
```

Comment:
```suggestion
In the three dots menu in the automation list or automation editor UI, select the **Run** button. This will execute all of the {% term actions %}, while skipping all {% term triggers %} and {% term conditions %}. This lets you test the full sequence of actions, as if the automation was triggered and all conditions were true. Note that any [trigger ID](/automation/trigger/#trigger-id) used in your triggers will not be active when you test this way. The Trigger ID or any data passed by in the `trigger` data in conditions or actions can't be tested directly this way.
```